util/file.go

```diff
@@ -378,12 +378,11 @@ func ExpandArguments(args []interface{}, folder string) (result []interface{}) {
 			if !filepath.IsAbs(arg) {
 				arg = prefix + arg
 			}
-			if expanded, _ := filepath.Glob(arg); expanded != nil {
-				for i := range expanded {
-					// We remove the prefix from the result as if it was executed directly in the folder directory
-					expanded[i] = strings.TrimPrefix(expanded[i], prefix)
-					result = append(result, expanded[i])
-				}
+			expanded, _ := filepath.Glob(arg)
```

Comment:
couldn't be nil?

Reply:
That's the cool thing about this. If it is nil, the range will simply do nothing, like an empty list. This is why the static check removes the nil condition